stratum/hal/lib/common/config_monitoring_service.h

```diff
@@ -60,6 +60,10 @@ class ConfigMonitoringService final : public ::gnmi::gNMI::Service {
                                    std::unique_ptr<ChassisConfig> config)
       LOCKS_EXCLUDED(config_lock_);
 
+  // Verifies platform independent properties of the given ChassisConfig proto.
+  // Note that PushChassisConfig calls VerifyChassisConfig at the beginning.
```

Comment:
We calls the `VerifyChassisConfig` before calling `PushChassisConfig`. But this comment shows one function calls another function.
How about `Note that VerifyChassisConfig will/must be called before PushChassisConfig`?

Reply:
It actually is called by the `ConfigMonitoringService::PushChassisConfig`: https://github.com/stratum/stratum/blob/32a71f141f9292f30dc4183c5c9007f876779aee/stratum/hal/lib/common/config_monitoring_service.cc#L113-L125

We also call it in `DoSet`, before handing the config to the `switch_interface_`.